drivers/flash/flash_stm32_qspi.c

```diff
@@ -20,6 +20,15 @@
 #include <drivers/dma.h>
 #include <drivers/dma/dma_stm32.h>
 
+#if DT_INST_NODE_HAS_PROP(0, spi_bus_width)
+#if DT_INST_PROP(0, spi_bus_width) == 4
+#define STM32_QSPI_QUAD_IO
+#endif
+#endif
+
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
```

Comment:
The `DT_INST_NODE_HAS_PROP(0, reset_gpios)` is used so often that maybe it deserves some specific define, that would clarify the intention.